demos/dotnet/dotnet/dotnet.cs

```diff
@@ -106,11 +111,14 @@ static void Build(string[] args, Log log)
         NugetAction.DownloadNugetAction(properties);
         NugetAction.RestorePackagesAction(properties, Log);
 
-        if (CscAction.Execute(properties, Log))
+        if (CscAction.Execute(properties, Log, buildLib))
         {
-            ConvertToCoreConsoleAction(properties);
+            if (!buildLib)
+            {
+                ConvertToCoreConsoleAction(properties);
+            }
             OutputRuntimeDependenciesAction(properties);
-            Console.WriteLine("bin\\{0}.exe created", properties.AssemblyName);
+            Console.WriteLine(buildLib ? "bin\\{0}.dll created" : "bin\\{0}.exe created", properties.AssemblyName);
```

Comment:
I would do:
("{0} created", properties.AssemblyNameAndExtension);
... and add the property to ProjectProperties